controllers/configurationpolicy_controller.go

```diff
@@ -606,18 +554,31 @@ func (r *ConfigurationPolicyReconciler) cleanUpChildObjects(plc policyv1.Configu
 			continue
 		}
 
-		namespaced := object.Object.Metadata.Namespace != ""
-
 		// determine whether object should be deleted
 		needsDelete := false
 
-		existing, err := getObject(
-			namespaced,
+		// Try from the cache first in case the evaluation interval of the ConfigurationPolicy is set to "watch".
+		// Note that GetFromCache does not start a watch if a watch wasn't present, unlike "Get".
+		existing, err := r.DynamicWatcher.GetFromCache(
+			gvk,
 			object.Object.Metadata.Namespace,
 			object.Object.Metadata.Name,
-			mapping.Resource,
-			r.TargetK8sDynamicClient,
 		)
+		if err != nil {
+			log.V(2).Error(
```

Comment:
I'm not sure what `.V(2).Error` will do combined... do you have an example output?

Reply:
I thought it changed the log level but still display the stack trace but that's not documented behavior from what I can tell, so I'll change it to an info log with the error message as a key/value.